mp/src/game/client/momentum/ui/menus/hud_menu_static.cpp

```diff
@@ -32,6 +32,12 @@ bool CHudMenuStatic::ShouldDraw() { return CHudElement::ShouldDraw() && m_bMenuD
 
 bool CHudMenuStatic::IsMenuDisplayed() { return m_bMenuDisplayed && m_bMenuTakesInput; }
 
+bool CHudMenuStatic::IsMenuDisplayed(const char *pszName)
+{
+    return IsMenuDisplayed() && FStrEq(pszName, m_pszFilename) ||
+           FStrEq(pszName, m_kvFromFile->GetString("menu_name", ""));
```

Comment:
Why not just use `m_kvFromFile->GetName()`? It'd save ya the effort of duplicating the menu name.

Reply:
very true!